Automattic/jetpack
# Podcast Player: Increase podcast shown

Fixes #21291

#### Changes proposed in this Pull Request:

* Created a new API to retrieve number of tracks.
* Added a method to call the above mentioned api
* Updating the max value of rangeControlUI

#### Jetpack product discussion NO



#### Does this pull request change what data or activity we track or use? NO



#### Testing instructions:





* Checkout to this PR 
* Build and run project
* Create/edit a post and add `podcast player` block
* Use the following link to embed https://distributed.blog/category/podcast/feed/
* Add the following code in current theme `functions.php` file
```php
function my_jetpack_podcast_helper_tracks_quantity( $tracks_quantity ) {
	return 42;
}
add_filter( 'jetpack_podcast_helper_tracks_quantity', 'my_jetpack_podcast_helper_tracks_quantity' );

```
* Reload the page and check `number of items` button in podcast settings UI.
* Change value instead `return 15;` of filter to some different value and it will update max limit in the rangeControlUI as well.




**Screenshots**
- Number of Items
<img width="361" alt="image" src="https://user-images.githubusercontent.com/21127788/140568143-71a1c132-4496-4b82-857d-5e96f52470a5.png">

cc @samiff , I have created a separate branch because there are a lot of  changes in the previous branch.
